templates/debops__tpl_macros.j2

```diff
@@ -129,3 +129,64 @@
 {%   endif %}
 {{ merged_dict | to_json }}
 {% endmacro %}{# ]]] #}
+
+{% macro flattened() %}{# [[[ #}
+{# This macro does what the flattened lookup from Ansible fails to do in Jinja templates as of Ansible 2.2.
+## All macro arguments are flattened into one "flat" list.
+## Uses a less known feature of Jinja for using the *args and *kwargs syntax as known from
+## Python. Even if the macro does not declare any arguments, it will happyily
+## flatten any non-key-value arguments you provide.
+## Additional key-value arguments can be used to influence the behavoir of the macro.
```

Comment:
That happens when my Vim does not have spell checking enabled for the current filetype and section for some reason :)